tests/test_input_doc.py

```diff
@@ -202,6 +237,6 @@ def _make_input_doc_from_stream(doc_stream):
         path_or_stream=doc_stream.stream,
         format=InputFormat.PDF,
         filename=doc_stream.name,
-        backend=PyPdfiumDocumentBackend,
+        backend=DoclingParseV4DocumentBackend,
```

Comment:
this is not producing GT test results
```suggestion
```

Reply:
The `InputDocument` __init__ has mandatory backend argument.

Reply:
Addressed.